Rafter won't compile on Erlang/OTP 19

Make process generates error:
```
src/rafter.erl: error in parse transform 'lager_transform': {function_clause,
                                             [{lager_transform,
                                               '-walk_ast/2-fun-0-',
                                               [{typed_record_field,
                                                 {record_field,5,
                                                  {atom,5,term}},
                                                 {type,5,non_neg_integer,[]}}],
                                               [{file,
                                                 "src/lager_transform.erl"},
                                                {line,53}]},
                                              {lists,map,2,
                                               [{file,"lists.erl"},
                                                {line,1239}]},
                                              {lager_transform,walk_ast,2,
                                               [{file,
                                                 "src/lager_transform.erl"},
                                                {line,53}]},
                                              {compile,
                                               '-foldl_transform/2-anonymous-2-',
                                               2,
                                               [{file,"compile.erl"},
                                                {line,958}]},
                                              {compile,foldl_transform,2,
                                               [{file,"compile.erl"},
                                                {line,960}]},
                                              {compile,
                                               '-internal_comp/4-anonymous-1-',
                                               2,
                                               [{file,"compile.erl"},
                                                {line,315}]},
                                              {compile,fold_comp,3,
                                               [{file,"compile.erl"},
                                                {line,341}]},
                                              {compile,internal_comp,4,
                                               [{file,"compile.erl"},
                                                {line,325}]}]}
Compiled src/rafter_backend_echo.erl
make: *** [rafter] Error 1
```

--- a/rebar.config
+++ b/rebar.config
@@ -4,6 +4,6 @@
 
 {erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform}, {platform_define, "^[1-9]", otp17orNewer}]}.
 
-{deps, [{lager, ".*", {git, "git://github.com/basho/lager", {tag, "2.0.1"}}},
+{deps, [{lager, ".*", {git, "git://github.com/basho/lager", {tag, "2.2.3"}}},
         {druuid, ".*", {git, "git://github.com/kellymclaughlin/druuid.git", {tag, "0.2"}}} 
 ]}.

